www/CocoaRestClient/Portfile

```diff
@@ -0,0 +1,51 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+name                CocoaRestClient
+
+github.setup        mmattozzi cocoa-rest-client 1.4.5
+github.tarball_from releases
+
+categories          www
+platforms           darwin
+supported_archs     x86_64
+license             BSD
+maintainers         {@lhaeger} openmaintainer
```

Comment:
Just to clarify regarding the email: we are not asking anyone to make the github email public.

However not all of our workflows work well with exclusively github approach. If there's a bug report on Trac, if buildbot sends failure reports about ports failing to build, ... we would prefer to still have some email to be able to occasionally reach the maintainers. It definitely doesn't need to be your primary  /  personal email, but some email over which one can eventually reach you ...

(Your email is already "public" in the commit message you made. You could use the same one in obfuscated form  here.)

Reply:
okidoki.